src/components/global/Task/Task.jsx

```diff
@@ -1,16 +1,81 @@
 import React, { useState } from "react";
+import { Draggable } from "react-beautiful-dnd";
 import RightChevron from "../../../assets/icons/RightChevron";
+import TaskBoardSampleData from "../TaskBoardSampleData";
 import Subtask from "./Subtask";
 
 function Task(props) {
-  const { name, checked, expanded, onClick, selected, subtasks } = props;
+  const {
+    id,
+    name,
+    index,
```

Comment:
This prop is only relevant for a task on the board view

Reply:
I've separated Task into Task and BoardTask. BoardTask is now the only one of the two components that has these props